src/Toolkit/File.php

```diff
@@ -139,6 +152,18 @@ public function hash(): string
         return md5($this->root);
     }
 
+    /**
+     * Sends an appropriate header for the asset
+     *
+     * @param bool $send
+     * @return \Kirby\Http\Response|string
+     */
+    public function header(bool $send = true)
+    {
+        $response = new Response('', $this->mime());
+        return $send === true ? $response->send() : $response;
+    }
```

Comment:
Would mean the code would need to look something like

```php
$response = new Response('', $this->mime());

if ($send !== true) {
    return $response;
}

$result = $response->send();
return empty($result) !== true ? $result : null;
```

IMO I am not sure if this is such a good change since it differs how other parts like `Response::send` work. In general, it makes sense but we probably should change it in more places then (and on a separate PR).

Reply:
The `$result` is always empty as it's the body that we explicitly pass as `''` in the first line. So the code would be this:

```php
$response = new Response('', $this->mime());

if ($send !== true) {
    return $response;
}

$response->send();
```

Reply:
I think this is the only place where `Response` is only used to send headers. So I don't think this behavior would interfere with other uses.

But I'm open about leaving this as it is.

Reply:
@lukasbestle So is your suggestion to return `null` or nothing (void)? Your initial comment and your code differ there. I'm fine with either.

Reply:
Definitely `void`. My comment to "return `null`" was misunderstandable. I just meant that we shouldn't return anything in this case (= `void`), which automatically results in a `null` value.

I've changed it. :)